src/ucp/core/ucp_worker.c

```diff
@@ -1823,6 +1824,10 @@ ucs_status_t ucp_worker_create(ucp_context_h context,
     ucs_conn_match_init(&worker->conn_match_ctx, sizeof(uint64_t),
                         &ucp_ep_match_ops);
     kh_init_inplace(ucp_worker_rkey_config, &worker->rkey_config_hash);
+    status = ucs_ptr_map_init(&worker->ptr_map);
+    if (status != UCS_OK) {
+        goto err_free;
```

Comment:
err_free assumes `ucs_strided_alloc_cleanup(&worker->ep_alloc);` will be called, while `ucs_strided_alloc_init` was not called yet

Reply:
fixed